### ISSUE TITLE
Double underscore directory name crashes

### ISSUE THREAD

**Issue by aplatypus** [TeamAmaze/AmazeFileManager#2614]
**Describe the bug**
I just renamed a directory with underscores in the name and clicked to cd to uthe directory.  Amaze crashed.  See below....

**To Reproduce**
1. Go to SD card
2. Click add folder 
3. Enter  name "_tmp"
4. SAVE
5. Tap menu press Rename
6. Enter "__tmp__"
7. Save
8. name saved
9. click on new folder
10. App crashed.

**Expected behavior**

I wanted to go into the new directory.

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: Galaxy A70
 - OS: Android 11
 - Rooted: No
 - Version: kernel 4.14

**Additional context**

## Issue explanation (write below this line)

Crash report....

## Exception
* __User Action:__ UI Error
* __Request:__ Application crash
* __Version:__ 3.5.3
* __OS:__ Linux Android 11 - 30
* __Device:__ a70q
* __Model:__ SM-A705YN
* __Product:__ a70qdo
<details><summary><b>Crash log </b></summary><p>

```
java.lang.NullPointerException: Attempt to invoke virtual method 'androidx.documentfile.provider.DocumentFile androidx.documentfile.provider.DocumentFile.findFile(java.lang.String)' on a null object reference
	at com.amaze.filemanager.filesystem.FileUtil.getDocumentFile(FileUtil.java:949)
	at com.amaze.filemanager.filesystem.files.FileUtils.scanFile(FileUtils.java:243)
	at com.amaze.filemanager.filesystem.files.FileUtils.lambda$scanFile$1(FileUtils.java:225)
	at com.amaze.filemanager.filesystem.files.-$$Lambda$FileUtils$nQFhhNVP8RswRigIhHzl4wuQz60.run(Unknown Source:4)
	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:923)

```
</details>
<hr>

**Comment by VishalNehra** [comment]
I can't reproduce this issue :/

**Comment by EmmanuelMess** [comment]
This was fixed on 3.6.0.
